### ISSUE TITLE
Posts are added twice

### ISSUE THREAD

**Issue by kiliw** [ryanwiemer/gatsby-starter-gcn#65]
Hey there and thank you very much for your nice starter. So far everything works with with contentful and the whole setup, the only thing I came across is that every post I create in contentufl is added twice. 
![image](https://user-images.githubusercontent.com/43745180/59159072-bd8f0280-8ac4-11e9-8dde-3ac536f65f55.png)

I checked the querys with the `GraphiQL` and figured out that each posts exists twice but a different `ID`

I tired googeling and digging around in contentful and so far I could not find any fix. Does someone has an idea why that happens?

Thank you very much.

**Comment by kiliw** [comment]
Well, I got it :) Sorry for creating and closing this issue directly.

**Solution** 
In contentful my space had two languages, *en* and *de* and both were send with the response. So there have been two different `IDs` for each post because of the languages.
I deleted the *de* version and now everything is fine.

In the case someone wants to have multilanguages there would be the possibility to add an filter for the `node_locale` and thus only selecting the necessary parts.

Anyway, thanks again for the nice starter.
